backend/src/routes/Registry/registry-controllers.ts

```diff
@@ -20,6 +21,17 @@ export const createRegistry: RequestHandler = async (req, res, next) => {
   }
 };
 
+export const getMyRegistry: AuthHandler = async (req, res, next) => {
+  try {
+    const userId = getUserId(req.user?.sub);
+    const registry = await Registry.findOne({ userId });
+    if (!registry) throw createError(500, "You don't have a registry");
```

Comment:
500 is a server error. Since the server processed the request, return a 404 not found